index.html

```diff
@@ -140,7 +140,7 @@ <h3>
         <div class="flexbox gray_border_top">
             <img class="small_icons_square" src="./Images/add_icon.PNG" alt="add icon">
             <p class="aling">Add to</p>
-            <img class="small_icons" src="./Images/share_icon1.PNG" alt="share icon">
+            <img class="small_icons" src="./Images/share_icon.PNG" alt="share icon">
```

Comment:
Hello team, the typo the last code reviewer was talking about is that you saved the icon as Share_icon.PNG, not share_icon.PNG.

Reply:
Hi Lucas, Thanks for everything! I was so blind, I couldn´t see what the problem was.